docker/GPU/Dockerfile

```diff
@@ -66,9 +66,11 @@ RUN git clone https://github.com/open-mmlab/mmdeploy &&\
     pip install -e .
 
 ### build sdk
-RUN git clone https://github.com/openppl-public/ppl.cv.git &&\
+RUN wget https://github.com/openppl-public/ppl.cv/archive/refs/tags/v0.6.1.zip && unzip v0.6.1.zip
+RUN mv ppl.cv-0.6.1 ppl.cv && \
```

Comment:
Hi, may you kindly build pplcv in single `RUN`?

Reply:
created another one seperately, my bad, new to git : )

Reply:
That's fine. We all make improvements through mistakes coping with new things. :smile: